lib/grpfree.gd

```diff
@@ -104,11 +104,12 @@ DeclareSynonym( "IsElementOfFreeGroupFamily",IsAssocWordWithInverseFamily );
 ##      The optional argument <A>name</A> must be a string; its default value is
 ##      <C>"f"</C>,
 ##      and the optional argument <A>init</A> must be a finite list of
-##      nonempty strings; its default value is an empty list.
+##      pairwise-distinct nonempty strings; its default value is an empty list.
```

Comment:
I'm not overly invested in this comment but I find the "pairwise" in this redundant and think it'd be nicer if it said "distinct"